### ISSUE TITLE
7.5.10 Errors AsyncPlayerChatEvent

### ISSUE THREAD

**Issue by LogGits** [kangarko/ChatControl-Pro#391]
Error when the users types anything in chat (I didnt /reload the server or use plugman. This happens when the server is restarted):
```
15:25:28Could not pass event AsyncPlayerChatEvent to ChatControl v7.5.10
Console 27/09/2017 15:25:28
15:25:28java.lang.NullPointerException: null
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_144]
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_144]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_144]
at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_144]
at me.kangarko.chatcontrol.null.lPt4.const(SourceFile:700) ~[?:?]
at me.kangarko.chatcontrol.null.cOM8.class(SourceFile:341) ~[?:?]
at me.kangarko.chatcontrol.null.cOM8.const(SourceFile:337) ~[?:?]
at me.kangarko.chatcontrol.null.nUL.const(SourceFile:261) ~[?:?]
at me.kangarko.chatcontrol.null.nUL.const(SourceFile:210) ~[?:?]
at me.kangarko.chatcontrol.null.nUL.const(SourceFile:163) ~[?:?]
at me.kangarko.chatcontrol.null.con.const(SourceFile:122) ~[?:?]
at me.kangarko.chatcontrol.null.con.const(SourceFile:118) ~[?:?]
at me.kangarko.chatcontrol.null.con.const(SourceFile:111) ~[?:?]
at me.kangarko.chatcontrol.null.com6.const(SourceFile:324) ~[?:?]
at me.kangarko.chatcontrol.null.Com3.const(SourceFile:49) ~[?:?]
at me.kangarko.chatcontrol.null.LpT5.const(SourceFile:91) ~[?:?]
at me.kangarko.chatcontrol.null.LpT5.const(SourceFile:52) ~[?:?]
at me.kangarko.chatcontrol.null.LpT5.const(SourceFile:29) ~[?:?]
at me.kangarko.chatcontrol.null.cOm1.execute(SourceFile:28) ~[?:?]
at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:74) ~[patched_1.12.2.jar:git-Paper-1220]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.12.2.jar:git-Paper-1220]
at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:514) [patched_1.12.2.jar:git-Paper-1220]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:496) [patched_1.12.2.jar:git-Paper-1220]
at net.minecraft.server.v1_12_R1.PlayerConnection.chat(PlayerConnection.java:1346) [patched_1.12.2.jar:git-Paper-1220]
at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1261) [patched_1.12.2.jar:git-Paper-1220]
at net.minecraft.server.v1_12_R1.PacketPlayInChat$1.run(PacketPlayInChat.java:39) [patched_1.12.2.jar:git-Paper-1220]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_144]
at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_144]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_144]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_144]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_144]
```
This has been happening since 7.5.9
Starts up fine no errors (even in log file).

**Comment by astirling** [comment]
I've been having similar issues since 7.5.8.  When I revert back to 7.5.7 everything works fine.
- Plugin loads with no errors
- Joining channels works fine (can see this with /channel)
- I can see Bungee connected chat from other servers
- When trying to type in the joined channel nothing shows up in game but an errors shows up in console
- When trying to send a message to a channel 'An unhandled error occured during executing the command.' message pops up in game chat and an error shows up in console.

Server is running latest paper spigot build 1220
I've updated essentialsx to the lastest snapshot build #23

[Server log](https://pastebin.com/BGmjvDEB) - This includes startup and errors from ChatControl Pro 7.5.10 (also seen in 7.5.8 and 7.5.9)
[Chatcontrol error log](https://pastebin.com/YK2ATZAX)

**Comment by kangarko** [comment]
Thank you for noticing. This has been resolved in 7.5.11
